input/blog/2018-08-21-cake-v0.30.0-released.md

```diff
@@ -0,0 +1,76 @@
+﻿---
+title: Cake v0.30.0 released
+category: Release Notes
+author: devlead
+---
+
+Version 0.30.0 of Cake has been released.
+
+With this release we've got several improvements and bug fixes contributed from the community, so firstly big _**thanks**_ to all for your continued support!
+
+In this post we would like to highlight that Cake is now available as a global .NET CLI tool.
+
+This means you can easily install Cake and have it globally available in your environment using the `dotnet tool` command.
+
+```
+dotnet tool install -g Cake.Tool --version 0.30.0
+```
+
+Once installed, then you can launch `cake` from anywhere using the .NET CLI
```

Comment:
Remove `then`: `Once installed, you can launch`

Reply:
addressed